index.js

```diff
@@ -0,0 +1,3 @@
+aaa = bbb;
+bbb = ccc;
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ccc' is not defined  no-undef